### DISCUSSION TITLE
SO 2.3.220 installation skips partitioning?

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#9989] [Unsupported Versions]
Hello!

I boot from the ISO image as a virtual CD/DVD on my new server and the SO installation process (heavynode installation for my existing grid) just skips the partitioning of my drives. Does that relate to https://github.com/Security-Onion-Solutions/securityonion/discussions/1791 or https://github.com/Security-Onion-Solutions/securityonion/discussions/9049 ? What now? Start all over, but what should I do diffenrently? Just tweek LVM manually to move /nsm to the right volume?

What else do I have to change manually beside the right location of /nsm?

Cheers, Ben

**Comment by InfosecGoon** [comment]
The installer should ask which mounted devices you want to put / and /nsm on -- if you just have one large storage device attached, it will put everything there but in separate LVM partitions.

**Comment by ben-sec** [comment]
Not in my case. It took my small boot drive (200 GB) and put LVM partitions on it and ignored my additional 10 TB Raid 5 volume. I had to move everything manually in the right place to make use of the 10 TB for /nsm. Why?

Von: Matthew Gracie ***@***.***>
Gesendet: Dienstag, 18. April 2023 16:39
An: Security-Onion-Solutions/securityonion ***@***.***>
Cc: Bruns, Benjamin ***@***.***>; Author ***@***.***>
Betreff: Re: [Security-Onion-Solutions/securityonion] SO 2.3.220 installation skips partitioning? (Discussion #9989)


The installer should ask which mounted devices you want to put / and /nsm on -- if you just have one large storage device attached, it will put everything there but in separate LVM partitions.

**Reply by InfosecGoon** [reply]
That's strange. Is it possible that the kernel in the installer didn't support your RAID controller?

**Reply by Acewiza** [reply]
Having /nsm setup the way you want it prior to SO install seems to work best for me.
